### PR TITLE
Support having multiple versions of Node in the repo at one time

### PR DESCRIPTION
**Pull request by SimenB**
This will make it trivial to update to new versions (currently 6.9.5 and 7.5.0)

### CODE REVIEW

**Comment by stigkj** [comment]
Looks nice! Not sure I understand why you need the `6/`and `7/`directories, though. Won't that be handled with a `release.sh <6.9.1/7.1.0>`?

**Comment by SimenB** [comment]
I could ignore them, but I find it nice to be able to go to a tag and see exactly what the Dockerfile was at the time. If you think it's unnecessary, I could remove them?

**Comment by SimenB** [comment]
@stigkj I updated to ignore the generated images 😄

File: Dockerfile.base [hunk 1]
```diff
@@ -9,11 +9,11 @@ EXPOSE 3000
 RUN mkdir -p /home/node/src
 WORKDIR /home/node/src
 
-RUN apk add --no-cache --virtual dumb-init-dependencies ca-certificates wget \
-	&& update-ca-certificates \
-	&& wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 \
-	&& chmod +x /usr/local/bin/dumb-init \
-	&& apk del dumb-init-dependencies
+RUN apk add --no-cache --virtual dumb-init-dependencies ca-certificates wget
```
Comment (SimenB) [inline Dockerfile.base]: I've upgraded the script to do `docker build --squash`, which means chaining a massive command together is no longer necesarry

**Comment by stigkj** [comment]
👍  You get the possibility to see what the Dockerfile was at a tag anyway, right?

File: Dockerfile.base (hunk 1 above)
Comment (SimenB) [inline Dockerfile.base]: This makes local builds go faster, since we ca cache on multiple levels

**Comment by SimenB** [comment]
Yeah, the template will be correct expect for the version number in the `FROM`

File: Dockerfile.base [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:6.9.1-alpine
+FROM node:0.0.0-alpine
```
Comment (theneva) [inline Dockerfile.base]: huh?

File: Dockerfile.onbuild [hunk 3]
```diff
@@ -1,16 +1,17 @@
-FROM finntech/node:6.9.1
+FROM finntech/node:0.0.0
```
Comment (theneva) [inline Dockerfile.onbuild]: ???

File: Dockerfile.base (hunk 2 above)
Comment (SimenB) [inline Dockerfile.base]: These are templates. The whole point of this PR is that those `0.0.0` are replaced by the version you want to release by the script

File: Dockerfile.onbuild (hunk 3 above)
Comment (SimenB) [inline Dockerfile.onbuild]: ditto

**Pull request merged**